vpt/sinks/file_store/__init__.py

```diff
@@ -0,0 +1,47 @@
+import os
+import contextlib
+from pathlib import Path
+from typing import Union
+
+import keyboard
+import mouse
+import numpy as np
+import soundfile as sf
+
+from vpt.sinks.base import SinkBase
+from vpt.sources.base import SourceBase
+
+
+class FileStore(SinkBase):
+    def __init__(self, dir_name: str, mouse_source: SourceBase,
+                 keyboard_source: SourceBase, audio_source: SourceBase):
+        super().__init__()
+        self.dir = Path(dir_name)
+
+        self.mouse_file = open(self.dir / 'mouse_output.txt', 'w+')
```

Comment:
Consider making the file unbuffered and remove invocations of `file.flush()` in the corresponding `_process_mouse` method.
```suggestion
        self.mouse_file = open(self.dir / 'mouse_output.txt', 'w+', buffered=0)
```